build/build.zig.zon

```diff
@@ -0,0 +1,22 @@
+.{
+    .name = "microzig/build",
+    .version = "0.0.0",
+    .dependencies = .{
+        .uf2 = .{
+            .path = "../tools/uf2",
+        },
+        .regz = .{
+            .path = "../tools/regz",
+        },
+        .@"microzig/core" = .{
+            .path = "../core",
+        },
+    },
+
+    .paths = .{
+        "README.md",
+        "build.zig",
+        "build.zig.zon",
+        "LICENSE",
+    },
```

Comment:
This is missing the `src/generate_linkerscript.zig` file, `README.md` does not exist, `LICENCE` is missing.

I guess we should perform a final cleanup in the end, and also *only* package files present in `.paths` to the website?